gensim/models/fasttext.py

```diff
@@ -469,77 +466,68 @@ def __init__(self, sentences=None, corpus_file=None, sg=0, hs=0, size=100, alpha
         if self.word_ngrams <= 1 and max_n == 0:
             bucket = 0
 
-        self.wv = FastTextKeyedVectors(size, min_n, max_n, bucket, compatible_hash)
-        self.vocabulary = FastTextVocab(
-            max_vocab_size=max_vocab_size, min_count=min_count, sample=sample,
-            sorted_vocab=bool(sorted_vocab), null_word=null_word, ns_exponent=ns_exponent)
-        self.trainables = FastTextTrainables(vector_size=size, seed=seed, bucket=bucket, hashfxn=hashfxn)
-        self.trainables.prepare_weights(hs, negative, self.wv, update=False, vocabulary=self.vocabulary)
-        self.wv.bucket = self.trainables.bucket
+        self.wv = FastTextKeyedVectors(vector_size, min_n, max_n, bucket, compatible_hash)
+        self.bucket = bucket
```

Comment:
Didn't choose this name, seems low priority, so didn't try to improve it. Improving it would break consistency with the Facebook FastText implementation, which uses exactly this name (singular, both in its command-line option & variable names).

Reply:
The FB implementation was done by another Czech, its choice of language is not something we necessarily have to follow. Although I guess it helps. Maybe.

Anyway low priority I guess, like you say.